### ISSUE TITLE
Intellisense suggestions is very slow with scikit learn package

### ISSUE THREAD

**Issue by YachaywasiAI** [microsoft/vscode#173230]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.75.0
- OS Version: PopOs 22.04 LTS

Steps to Reproduce:
1. Clean installation of VSCode, with only Material Icon and the last Python extensions.
1. Miniconda environment with packages Scikit-learn 1.12, Pytorch 1.13.1
2. The following code causes slow suggestions to intellisense (specific when I import scikit-learn package)
3. https://gist.github.com/YachaywasiAI/1ac6b0511f828b487de4ef25f1eab0ea
4. When I type last line, the parameters inside of function roc_auc_score. The time to suggest word "labels" or "predictions" is high.

**Comment by pindjur** [comment]
Same here but different package (@stitches/react). Intellisense not working. Showing "...loading" indefinetly. It works for other files when i close all files and reload window, but when i open that one file file which contains import to stitches package, then intellisense doesn't work for all files. 

I also tried reload eith extensions disabled, pc restart with no luck.
Happend after updating vscode 30min ago.

**Comment by VSCodeTriageBot** [comment]
It looks like this is caused by the Python extension. Please file the issue to the [Python extension repository](https://github.com/microsoft/vscode-python). Make sure to check their issue reporting template and provide them relevant information such as the extension version you're using. See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting) for more information.

Happy Coding!
